psl-parser/src/main/java/org/linqs/psl/parser/CommandLineLoader.java

```diff
@@ -378,6 +372,8 @@ public int compare(Option o1, Option o2) {
      */
     private static CommandLine parseOptions(String[] args) {
         Options options = setupOptions();
```

Comment:
Don't shadow if you can avoid it (this shadows the static `CommandLine` member).